content/baremetal/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Baremetal"
+date: 2019-06-18T16:42:20+02:00
+lastmod: 2019-06-18T16:42:20+02:00
+publishdate: 2019-06-18T16:42:20+02:00
+draft: false
+weight: 23
+---
+
+# unblock a stuck `oc delete bmh` command
+
+Sometimes, when deleting a BMH object with `oc delete bmh -n openshift-machine-api <node name>`
+the delete command is stuck forever.
+This happens because ironic is trying to decommision and delete stuff from the node itself, and does not always succed with that.
```

Comment:
```suggestion
This happens because ironic is trying to decommision and delete stuff from the node itself, and does not always succeed with that.
```